boilerplate/flyte/end2end/run-tests.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python3
```

Comment:
This script will be copied from `flytetools` in the final version, right? I'm not sure how boilerplate works, so I'm asking.

Reply:
yes i used the same script

Reply:
sorry, I should have been more clear. How are these files under `/boilerplate` updated?

Reply:
Followed up offline. We use a workflow to update boilerplate in each repo.